components/Card.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <NuxtLink
-    class="card w-full p-2 rounded-xl shadow-sm border border-color-outline-transparent cursor-pointer hover:bg-interactive-tertiary-hover hover:shadow-md focus:ring-1 focus:ring-offset-2 focus:ring-focus-ring"
+    class="not-prose card w-full p-2 rounded-xl shadow-sm border border-color-outline-transparent cursor-pointer hover:bg-interactive-tertiary-hover hover:shadow-md focus:ring-1 focus:ring-offset-2 focus:ring-focus-ring"
     :to="props.link"
   >
     <div class="p-2 space-y-2">
       <component
         :is="currentIcon"
         class="w-6 h-6 fill-brand-accent stroke-brand-accent"
       />
-      <h2>
+      <h2 class="type-headline-4">
```

Comment:
@MeganSteenkamp I'm unfamiliar with these - where do these `type-*` styles come from?

Reply:
@tom-blake these are defined in the `typography.js` file - they align with the class names use in Figma.